src/components/result/FilterSelect/types.ts

```diff
@@ -1,7 +1,11 @@
 import type { SelectOnChangeHandler } from '@offer-ui/react'
 import type { ChangeEventHandler } from 'react'
+import type { GetPostsRes } from '@apis/post'
+import type { ApplyPriceType } from '@hooks/result/useSelectBoxFilter'
```

Comment:
```suggestion
import type { GetPostsRes } from '@apis'
import type { ApplyPriceType } from '@hooks'
```
베럴 파일 활용되면 좋을 것 같습니다!

Reply:
[완료!](https://github.com/price-offer/offer-fe/pull/174/commits/a6f681de442a40533851b770a8a71280628c236f#diff-4753ae1ab8636d729c9f6593563402bb609e1919a773ca36d0a37196b5a3235b)